deploy/helm/flightctl/templates/flightctl-server-deployment.yaml

```diff
@@ -25,6 +25,8 @@ spec:
           env:
             - name: HOME
               value: "/root"
+            - name: FLIGHTCTL_DISABLE_AUTH
```

Comment:
i've disabled auth until we have a default (keycloak) auth setup.
Otherwise deployment e2e tests would fail (or any other deployment not using k8s auth)